.pipelines/cni/pipeline.yaml

```
@@ -153,6 +169,34 @@ stages:
       vmSize: ${VM_SIZE}
       arch: amd64
 
+  - template: singletenancy/linux-cniv2-template.yaml
+    parameters:
+      name: mariner_linux_overlay
+      clusterType: overlay-byocni-up
+      clusterName: mariner-cniv2-overlay
+      nodeCount: ${LINUX_NODE_COUNT}
+      vmSize:  ${LINUX_VM_SKU}
+      arch: amd64
+      osSKU: AzureLinux
+
+  - template: singletenancy/linux-cniv2-template.yaml
+    parameters:
+      name: arm_linux_overlay
+      clusterType: overlay-byocni-up
+      clusterName: arm-cniv2-overlay
+      nodeCount: ${LINUX_NODE_COUNT}
+      vmSize: Standard_D8ps_v5
+      arch: arm64

```

Lets match these to with the templates above `${LINUX_NODE_COUNT}` -> `${NODE_COUNT_LINUX}` ect.
```suggestion
      clusterType: overlay-byocni-up
      clusterName: mariner-cniv2-overlay
      nodeCount: ${NODE_COUNT_LINUX}
      vmSize:  ${VM_SIZE}
      arch: amd64
      osSKU: AzureLinux

  - template: singletenancy/linux-cniv2-template.yaml
    parameters:
      name: arm_linux_overlay
      clusterType: overlay-byocni-up
      clusterName: arm-cniv2-overlay
      nodeCount: ${NODE_COUNT_LINUX}
      vmSize: Standard_D8ps_v5
      arch: arm64

```